utils/nctl/sh/scenarios-upgrades/upgrade_scenario_09.sh

```diff
@@ -69,11 +69,11 @@ function _step_01()
     source "$NCTL/sh/node/start.sh" node=all
 }
 
-# Step 02: Await era-id >= 1.
+# Step 02: Await era-id >= 2.
```

Comment:
You forgot to update the description as well https://github.com/casper-network/casper-node/pull/2791/files#diff-9528249f4209e92c90ded7f514e7addda46f995a463d0c94c117609d5b9eed8dR7